Modules/MUON/MCH/src/TracksTask.cxx

```diff
@@ -115,10 +115,6 @@ void TracksTask::initialize(o2::framework::InitContext& /*ic*/)
 {
   ILOG(Info, Support) << "initialize TracksTask" << ENDM; // QcInfoLogger is used. FairMQ logs will go to there as well.
 
-  if (!o2::base::GeometryManager::isGeometryLoaded()) {
-    TaskInterface::retrieveConditionAny<TObject>("GLO/Config/Geometry");
-  }
-
   createTrackHistos();
```

Comment:
Do we also need the `ctx.inputs().get<TGeoManager*>("mchgeo");` statement in the `monitorData()` method of `o2::quality_control_modules::muonchambers::TracksTask`?

Reply:
hum, yes, we probably should. Assuming we still want to use that Task (see other comment/question).